_posts/2024-04-12-a-time-machine-for-your-data-opensearch-searchable-snapshots.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "A time machine for your data: OpenSearch searchable snapshots"
+authors:
+  - abunday
+date: 2024-04-10
+categories:
+  - technical-post
+  - feature
+  - partners
+meta_keywords:
+meta_description:
```

Comment:
Please add the following meta: 

_**Meta keywords:**_ searchable snapshots, OpenSearch searchable snapshots, Instaclustr Managed Platform, remote snapshot index, Instaclustr OpenSearch clusters

**_Meta_description:_** Learn how Instaclustr by NetApp implemented searchable snapshots to efficiently store and search massive amounts of data and why they believe it can help organizations using OpenSearch do the same.

Reply:
Thanks! These have been added 👍